{
    "type": "resources",
    "link": [],
    "resource": [
        {
            "link": [],
            "content": {
                "type": "model-core.lib.codata.error.coErrorDto",
                "coErrorDto": {
                    "reference": "593564387",
                    "message": "Row for given parameters already exists.",
                    "requestUrl": null,
                    "httpCode": "422",
                    "errorType": "Error",
                    "imageUrl": "https://qline.tugraz.at/qtug/img/msg_error.gif"
                }
            }
        }
    ]
}